### PR TITLE
fixed error message for GatedRepoError

### PR DESCRIPTION
**Pull request by DawiAlotaibi**
#### Context
What is the purpose of this PR? Is it to
- [ ] add a new feature
- [ ] fix a bug
- [ ] update tests and/or documentation
- [x] Cleanup

Please link to any issues this PR addresses.
https://github.com/pytorch/torchtune/issues/1831

#### Changelog
What are the changes made in this PR?
Checked if `args.hf_token` is passed or not. and printed a message accordingly.

#### Test plan
None

- [x] run pre-commit hooks and linters (make sure you've first installed via `pre-commit install`)
- [ ] add [unit tests](https://github.com/pytorch/torchtune/tree/main/tests/torchtune) for any new functionality
- [ ] update [docstrings](https://github.com/pytorch/torchtune/tree/main/docs/source) for any new or updated methods or classes
- [x] run unit tests via `pytest tests`
- [x] run recipe tests via `pytest tests -m integration_test`
- [ ] manually run any new or modified recipes with sufficient proof of correctness
- [ ] include relevant commands and any other artifacts in this summary (pastes of loss curves, eval results, etc.)

#### UX
If your function changed a public API, please add a dummy example of what the user experience will look like when calling it.
Here is a [docstring example](https://github.com/pytorch/torchtune/blob/6a7951f1cdd0b56a9746ef5935106989415f50e3/torchtune/modules/vision_transformer.py#L285)
and a [tutorial example](https://pytorch.org/torchtune/main/tutorials/qat_finetune.html#applying-qat-to-llama3-models)

- [x] I did not change any public API
- [ ] I have added an example to docs or docstrings

### CODE REVIEW

**Review by joecummings — COMMENTED** [review]
Thanks for this fix @DawiAlotaibi! Two things:
1. Would you mind updating the appropriate test for this change?
2. Can you sign the CLA?

**Comment by DawiAlotaibi** [comment]
> Thanks for this fix @DawiAlotaibi! Two things:
> 
> 1. Would you mind updating the appropriate test for this change?
> 2. Can you sign the CLA?

Added the test cases please let me know if they are appropriate. For the CLA I am waiting on confirmation to sign it and it might take some time, will let you know once I have signed it.

**Review by joecummings — APPROVED** [review]
This looks great! Thanks for such a quick fix - will merge when CI is green

**Pull request merged**
